exercise/templates/exercise/_user_results.html

```diff
@@ -16,22 +19,37 @@
 </p>
 {% endif %}
 
-<p id="toggle-expired" class="hide">
-	<button class="aplus-button--secondary aplus-button--xs">
-		<span class="glyphicon glyphicon-time" aria-hidden="true"></span>
-		{% translate "SHOW_OLDER_ASSIGNMENTS" %}
+<p>
+	<button id="toggle-expand-all-modules" class="aplus-button--secondary aplus-button--xs" style="display:none">
+		<span class="glyphicon glyphicon-unchecked" aria-hidden="true"></span>
+		{% translate "EXPAND_ALL_MODULES" %}
 	</button>
+	<span id="cycle-auto-scroll-behavior">
+		<button id="auto-scroll-behavior-smooth" class="aplus-button--secondary aplus-button--xs" style="display:none">
+			<span class="glyphicon glyphicon-check" aria-hidden="true"></span>
+			{% translate "AUTO_SCROLL_BEHAVIOR_SMOOTH" %}
+		</button>
+		<button id="auto-scroll-behavior-instant" class="aplus-button--secondary aplus-button--xs" style="display:none">
+			<span class="glyphicon glyphicon-flash" aria-hidden="true"></span>
+			{% translate "AUTO_SCROLL_BEHAVIOR_INSTANT" %}
+		</button>
+		<button id="auto-scroll-behavior-off" class="aplus-button--secondary aplus-button--xs" style="display:none">
+			<span class="glyphicon glyphicon-unchecked" aria-hidden="true"></span>
+			{% translate "AUTO_SCROLL_BEHAVIOR_OFF" %}
+		</button>
+	</span>
```

Comment:
It doesn't seem right to have a simple button that cycles through different options (the scroll behaviour button). I'm not sure what is the best way to do it but the two options I came up with are (1) two checkbox buttons: one for on/off and one for instant/smooth (hide/show the second button based on the first), and (2) three radio buttons (Does this fit the A+ style? Are they used anywhere?).

Reply:
Fixed by adding a separate button for enabling/disabling auto-scrolling and setting the scroll behavior.